app/controllers/change_requests_controller.rb

```diff
@@ -53,6 +65,68 @@ def create_change_request
     render status: :created, json: ChangeRequestsPresenter.present(@change_request)
   end
 
+  def handle_insert
+
+    change_request = params[:change_request]
+    if change_request[:type] == "addresses"
+      address = Address.new
+      addressattention = change_request[:field_changes][:attention]
+      address.city = change_request[:field_changes][:city]
+      address.address_1 = change_request[:field_changes][:address_1]
+      address.address_2 = change_request[:field_changes][:address_2]
+      address.address_3 = change_request[:field_changes][:address_3]
+      address.address_4 = change_request[:field_changes][:address_4]
+      address.city = change_request[:field_changes][:city]
+      address.state_province = change_request[:field_changes][:state_province]
+      address.postal_code = change_request[:field_changes][:postal_code]
+      address.country = change_request[:field_changes][:country]
+      address.online = change_request[:field_changes][:online]
+      address.region = change_request[:field_changes][:region]
+      address.name = change_request[:field_changes][:name]
+      address.description = change_request[:field_changes][:description]
+      address.transportation = change_request[:field_changes][:transportation]
```

Comment:
I think most of these could be done in a single, bulk statement using `assign_attributes`: https://stackoverflow.com/a/9751488